# Significance

Increased bacterial resistance to conventional antibiotics is well documented and leads to over two million infections and 23,000 deaths in the United States each year. [[Grant Proposal And Notes#1. Centers for Disease Control and Prevention. Antibiotic resistance threats in the United States; 2013. |1]] There is also a significant financial cost to these infections, of greater than $2 billion annually.[[Grant Proposal And Notes#2. Shrestha, P.; Cooper, B. S.; Coast, J.; Oppong, R.; Do Thi Thuy, N.; Phodha, T.; Celhay, O.; Guerin, P. J.; Wertheim, H.; Lubell, Y., Enumerating the economic cost of antimicrobial resistance per antibiotic consumed to inform the evaluation of interventions affecting their use. Antimicrob Resist Infect Control 2018, 7, 98.| 2]], [[Grant Proposal And Notes#3. Thorpe, K. E.; Joski, P.; Johnston, K. J., Antibiotic-Resistant Infection Treatment Costs Have Doubled Since 2002, Now Exceeding $2 Billion Annually. Health Aff (Millwood) 2018, 37 (4), 662-669.|3 ]]Concerns about bacterial resistance are compounded by the relative slowdown in the development and approval of novel therapeutics in recent decades.[[Grant Proposal And Notes#4. World Health Organization. Antibacterial agents in clinical and preclinical development an overview and analysis; 2021.| 4 ]] Antimicrobial peptides (AMPs) represent a promising avenue to address this clear need for novel antibiotics. While AMPs are not entirely immune to bacterial resistance, their broad targeting of biomolecular families makes them less susceptible to resistance mechanisms. [[Grant Proposal And Notes#5. Nizet, V., Antimicrobial peptide resistance mechanisms of human bacterial pathogens. Curr Issues Mol Biol 2006, 8 (1), 11-26.|5]]-[[Grant Proposal And Notes#7. Zasloff, M., Antimicrobial peptides of multicellular organisms. Nature 2002, 415 (6870), 389-95.|7]] AMPs also have potential to enhance the activity of other antibiotics in combination therapies. [[Grant Proposal And Notes#8. Grassi, L.; Maisetta, G.; Esin, S.; Batoni, G., Combination Strategies to Enhance the Efficacy of Antimicrobial Peptides against Bacterial Biofilms. Front Microbiol 2017, 8, 2409.|8]]-[[Grant Proposal And Notes#12. Zharkova, M. S.; Orlov, D. S.; Golubeva, O. Y.; Chakchir, O. B.; Eliseev, I. E.; Grinchuk, T. M.; Shamova, O. V., Application of Antimicrobial Peptides of the Innate Immune System in Combination With Conventional Antibiotics-A Novel Way to Combat Antibiotic Resistance? Front Cell Infect Microbiol 2019, 9, 128.|12]] Despite the promise of AMPs, there is a clear need to develop approaches to more effectively engineer peptides with enhanced activity. Although previous work identified a significant number of AMPs that are believed to bind to nucleic acids as part of their antimicrobial mechanism, [[Grant Proposal And Notes#13. Ghosh, A.; Kar, R. K.; Jana, J.; Saha, A.; Jana, B.; Krishnamoorthy, J.; Kumar, D.; Ghosh, S.; Chatterjee, S.; Bhunia, A., Indolicidin targets duplex DNA structural and mechanistic insight through a combination of spectroscopy and microscopy. ChemMedChem 2014, 9 (9), 2052-8.|13]]-[[Grant Proposal And Notes#29. Uyterhoeven, E. T.; Butler, C. H.; Ko, D.; Elmore, D. E., Investigating the nucleic acid interactions and antimicrobial mechanism of buforin II. FEBS Lett. 2008, 582 (12), 1715-1718.|29]] little previous research has utilized those interactions as a starting point for developing more active peptides. In this proposed work, we will use an integrated approach of computational design and experimental analyses to rationally design AMPs based on their DNA interactions. Moreover, we will focus on the need to consider potential interactions with lipid membranes that are critical for peptide function along with DNA interactions in our design strategy. In addition to providing a novel approach to AMP design, our results will also help elucidate some fundamental questions related to the structure-function relationships of AMPs. For example, our work will further confirm the role of DNA binding in peptide mechanisms and will consider how membrane interactions relate to the ability of peptides to permeabilize and/or translocate membranes. Our proposed studies will also consider the balance between membrane and DNA interactions in mechanisms of action. <mark style="background: #ABF7F7A6;">Thus, in addition to promoting the design of more potent AMPs that target DNA our proposed work will also provide more generalizable insight into the balance between membrane and intracellular target interactions that can be applied to the broader design of AMPs.</mark> Finally, this work will provide robust training opportunities for diverse cohorts of undergraduates consistent with the goals of the R15/AREA program.
# Innovation

A primary innovation of our proposed research will be the implementation of a design approach for AMPs that brings together molecular dynamics simulations and electrostatics-based charge optimization. To our knowledge, this approach has not been utilized in the design of more active AMPs. Moreover, we have chosen to apply this approach to AMPs hypothesized to interact with nucleic acids as part of their mechanism of action. If effective, our proposed approach could also be adapted not only as a framework for the broad-range development of active AMPs with nucleic acid targets but also for peptides that have other intracellular targets. Our design work will also be tightly integrated with experimental evaluation of peptides using a combination of biochemical measurements and bacterial assays. This integration will allow for us to effectively evaluate and refine our design process. The PI and co-investigator have an established track record of collaborative work effectively implementing this integrated approach to investigate structure-function relationships of AMPs, [[Grant Proposal And Notes#30. Perez, C. P.; Elmore, D. E.; Radhakrishnan, M. L., Computationally Modeling Electrostatic Binding Energetics in a Crowded, Dynamic Environment Physical Insights from a Peptide-DNA System. J. Phys. Chem. B 2019, 123 (50), 10718-10734.|30]], [[Grant Proposal And Notes#31. Sim, S.; Wang, P.; Beyer, B. N.; Cutrona, K. J.; Radhakrishnan, M. L.; Elmore, D. E., Investigating the nucleic acid interactions of histone-derived antimicrobial peptides. FEBS Lett 2017, 591 (5), 706-717.|31]] and the proposed studies build clearly upon our previous successes.
# Approach
[[Grant Proposal And Notes# |  ]]
## Background

Traditionally, AMPs were considered to primarily kill bacterial cells through disruption of the lipid membrane. [[Grant Proposal And Notes#32. Jenssen, H.; Hamill, P.; Hancock, R. E., Peptide antimicrobial agents. Clin. Microbiol. Rev. 2006, 19 (3), 491-511.|32]], [[Grant Proposal And Notes#33. Yeaman, M. R.; Yount, N. Y., Mechanisms of antimicrobial peptide action and resistance. Pharmacol. Rev. 2003, 55 (1), 27-55.|33]] However, researchers have increasingly identified AMPs that translocate into bacterial cells without causing significant membrane damage and instead target an intracellular molecule in their mechanism of action. [[Grant Proposal And Notes#34. Hale, J. D.; Hancock, R. E., Alternative mechanisms of action of cationic antimicrobial peptides on bacteria. Expert Rev Anti Infect Ther 2007, 5 (6), 951-9.|34 ]]A number of these studies have identified nucleic acids, particularly DNA, as the likely intracellular target for those AMPs. For example, the Antimicrobial Peptide Database (APD3)[[Grant Proposal And Notes#35. Wang, G.; Li, X.; Wang, Z., APD3 the antimicrobial peptide database as a tool for research and education. Nucleic Acids Res 2016, 44 (D1), D1087-93.| 35]] includes information on 17 unique peptides from different sources that have DNA interactions as part of their proposed mechanism of action.[[Grant Proposal And Notes#13. Ghosh, A.; Kar, R. K.; Jana, J.; Saha, A.; Jana, B.; Krishnamoorthy, J.; Kumar, D.; Ghosh, S.; Chatterjee, S.; Bhunia, A., Indolicidin targets duplex DNA structural and mechanistic insight through a combination of spectroscopy and microscopy. ChemMedChem 2014, 9 (9), 2052-8.| 13]]-[[Grant Proposal And Notes#27. Park, C. B.; Kim, H. S.; Kim, S. C., Mechanism of action of the antimicrobial peptide buforin II buforin II kills microorganisms by penetrating the cell membrane and inhibiting cellular functions. Biochem. Biophys. Res. Commun. 1998, 244 (1), 253-7.|27]] Moreover, previous research in the co-investigator’s lab has identified novel histone-derived antimicrobial peptides (HDAPs) that appear to target nucleic acids.[[Grant Proposal And Notes#28. Tsao, H. S.; Spinella, S. A.; Lee, A. T.; Elmore, D. E., Design of novel histone- derived antimicrobial peptides. Peptides 2009, 30 (12), 2168-73.| 28]], [[Grant Proposal And Notes#36. Pavia, K. E.; Spinella, S. A.; Elmore, D. E., Novel histone-derived antimicrobial peptides use different antimicrobial mechanisms. Biochim Biophys Acta 2012, 1818 (3), 869-76.|36]]

Perhaps the best studied DNA-targeting AMP is buforin II (BF2). A series of studies elucidated the BF2 mechanism of action, including its ability to translocate across membranes of both bacterial cells and lipid vesicles.[[Grant Proposal And Notes#27. Park, C. B.; Kim, H. S.; Kim, S. C., Mechanism of action of the antimicrobial peptide buforin II buforin II kills microorganisms by penetrating the cell membrane and inhibiting cellular functions. Biochem. Biophys. Res. Commun. 1998, 244 (1), 253-7.| 27]],[[Grant Proposal And Notes#29. Uyterhoeven, E. T.; Butler, C. H.; Ko, D.; Elmore, D. E., Investigating the nucleic acid interactions and antimicrobial mechanism of buforin II. FEBS Lett. 2008, 582 (12), 1715-1718.| 29]],[[Grant Proposal And Notes#37. Kobayashi, S.; Chikushi, A.; Tougu, S.; Imura, Y.; Nishida, M.; Yano, Y.; Matsuzaki, K., Membrane translocation mechanism of the antimicrobial peptide buforin 2. Biochemistry 2004, 43, 15610-15616.| 37]]-[[Grant Proposal And Notes#40. Xie, Y.; Fleming, E.; Chen, J. L.; Elmore, D. E., Effect of proline position on the antimicrobial mechanism of buforin II. Peptides 2011, 32 (4), 677-82.|40]] Moreover, this peptide is not only known to bind to DNA, but there is also a clear correlation between DNA binding and antimicrobial activity in a range of BF2 variants.[[Grant Proposal And Notes#27. Park, C. B.; Kim, H. S.; Kim, S. C., Mechanism of action of the antimicrobial peptide buforin II buforin II kills microorganisms by penetrating the cell membrane and inhibiting cellular functions. Biochem. Biophys. Res. Commun. 1998, 244 (1), 253-7.| 27]],[[Grant Proposal And Notes#29. Uyterhoeven, E. T.; Butler, C. H.; Ko, D.; Elmore, D. E., Investigating the nucleic acid interactions and antimicrobial mechanism of buforin II. FEBS Lett. 2008, 582 (12), 1715-1718.| 29]],[[Grant Proposal And Notes#31. Sim, S.; Wang, P.; Beyer, B. N.; Cutrona, K. J.; Radhakrishnan, M. L.; Elmore, D. E., Investigating the nucleic acid interactions of histone-derived antimicrobial peptides. FEBS Lett 2017, 591 (5), 706-717.| 31]] For many other peptides believed to target DNA, the link between DNA binding and activity is more indirect and is based primarily on the ability of a peptide bind DNA in vitro. While that binding might relate to an AMP’s mechanism of action, it could also be a circumstantial property that arises from the generally cationic character of these peptides. However, other studies have demonstrated more direct relationships between the presence of an AMP on DNA replication or DNA damage repair mechanisms.[[Grant Proposal And Notes#15. Libardo, M. D. J.; Bahar, A. A.; Ma, B.; Fu, R.; McCormick, L. E.; Zhao, J.; McCallum, S. A.; Nussinov, R.; Ren, D.; Angeles-Boza, A. M.; Cotten, M. L., Nuclease activity gives an edge to host-defense peptide piscidin 3 over piscidin 1, rendering it more effective against persisters and biofilms. FEBS J 2017, 284 (21), 3662-3683.| 15]], [[Grant Proposal And Notes#22. Zhang, K.; Zhang, H.; Gao, C.; Chen, R.; Li, C., Antimicrobial Mechanism of pBD2 against Staphylococcus aureus. Molecules 2020, 25 (15).|22]],[[Grant Proposal And Notes#23. Hansmann, B.; Schroder, J. M.; Gerstel, U., Skin-Derived C-Terminal Filaggrin-2 Fragments Are Pseudomonas aeruginosa-Directed Antimicrobials Targeting Bacterial Replication. PLoS Pathog 2015, 11 (9), e1005159.| 23]]

Having a known or hypothesized intracellular target opens up clear avenues for rational design of these AMPs. For example, one would hypothesize that AMPs that bind DNA more tightly would show enhanced activity, as demonstrated by the results shown for our preliminary design work with BF2 in Specific Aim 1 below. However, it is also clear that mutations designed to enhance DNA affinity may also alter membrane interactions that are critical to cellular entry. Thus, an effective design approach will need to consider the interplay between optimizing both the membrane and DNA interactions. This interplay will be particularly important to consider since there is also evidence that some peptides, including the DNA-binding peptides BF2 and DesHDAP1, might utilize a combination of membrane- active and intracellular-target based interactions in their mechanisms of action.[[Grant Proposal And Notes#41. Wei, L.; LaBouyer, M. A.; Darling, L. E.; Elmore, D. E., Bacterial Spheroplasts as a Model for Visualizing Membrane Translocation of Antimicrobial Peptides. Antimicrob Agents Chemother 2016, 60 (10), 6350-2.| 41]]

Here, we propose a computational design strategy that aims to enhance the antimicrobial potency of AMPs by optimizing their DNA binding and membrane interactions (Fig. 1). <mark style="background: #ABF7F7A6;">We hypothesize that the electrostatically-governed interactions between peptides and these targets may be enhanced by increasing their monopoles with one or more rationally-placed arginine residues.</mark> We will use arginine instead of lysine residues because previous work in our labs suggested that arginine mutants yielded greater DNA binding affinity, membrane interactions, and potency compared to lysine mutants.[[Grant Proposal And Notes#31. Sim, S.; Wang, P.; Beyer, B. N.; Cutrona, K. J.; Radhakrishnan, M. L.; Elmore, D. E., Investigating the nucleic acid interactions of histone-derived antimicrobial peptides. FEBS Lett 2017, 591 (5), 706-717.| 31]], [[Grant Proposal And Notes#42. Cutrona, K. J.; Kaufman, B. A.; Figueroa, D. M.; Elmore, D. E., Role of arginine and lysine in the antimicrobial mechanism of histone-derived antimicrobial peptides. FEBS Lett 2015, 589 (24 Pt B), 3915-20.|42]] Our design approach will utilize molecular dynamics (MD) simulations to determine structural ensembles for peptide-DNA or peptide-membrane interactions. These ensembles will then be used in an electrostatics-based charge optimization approach to predict mutations that will enhance either DNA or membrane interactions. Notably, we recognize that it is possible that the highest activity might arise from peptides that have a moderate—i.e. not too weak or too strong— interaction with targets, particularly in terms of membrane interactions. In turn, the molecular interactions of these predicted mutations will be considered through subsequent rounds of MD simulations and electrostatics calculations. The DNA and membrane interactions as well as the antibacterial activity of these predicted mutations will then be evaluated experimentally. The results of these experiments will both help us to assess the effectiveness of our design approach and address critical biological questions related to the role of DNA binding and membrane interactions in the mechanisms of action for these peptides. After performing optimizations for DNA binding and membrane binding separately, we will culminate this work by developing, applying, and analyzing processes to co-optimize peptide binding affinity toward both DNA and membrane to create engineered peptides that best balance those properties.

![[Pasted image 20240531175238.png]]
  Figure 1: Schematic of mechanism of action for AMP (gray) that translocates across the membrane and interacts with intracellular DNA. Our design strategy will focus on using MD simulations and electrostatic charge optimization methods to first optimize DNA binding (Aim 1) and membrane interactions (Aim 2) in isolation. Aim 3 will involve a co-optimization of both DNA and membrane interactions.

In order to choose the peptides to use for our initial design approach, we began with the set of AMPs that have hypothesized DNA targets in APD3[[Grant Proposal And Notes#13. Ghosh, A.; Kar, R. K.; Jana, J.; Saha, A.; Jana, B.; Krishnamoorthy, J.; Kumar, D.; Ghosh, S.; Chatterjee, S.; Bhunia, A., Indolicidin targets duplex DNA structural and mechanistic insight through a combination of spectroscopy and microscopy. ChemMedChem 2014, 9 (9), 2052-8.| 13]]-[[Grant Proposal And Notes#27. Park, C. B.; Kim, H. S.; Kim, S. C., Mechanism of action of the antimicrobial peptide buforin II buforin II kills microorganisms by penetrating the cell membrane and inhibiting cellular functions. Biochem. Biophys. Res. Commun. 1998, 244 (1), 253-7.|27]] as well as the DesHDAP peptides designed in the co-investigator’s lab.[[Grant Proposal And Notes#28. Tsao, H. S.; Spinella, S. A.; Lee, A. T.; Elmore, D. E., Design of novel histone- derived antimicrobial peptides. Peptides 2009, 30 (12), 2168-73.| 28]],[[Grant Proposal And Notes#36. Pavia, K. E.; Spinella, S. A.; Elmore, D. E., Novel histone-derived antimicrobial peptides use different antimicrobial mechanisms. Biochim Biophys Acta 2012, 1818 (3), 869-76.| 36]] We then performed BLAST searches of these peptides sequences against all experimentally solved structures in the PDB to determine whether there were potential template structures to use as starting points for peptide-DNA interactions. Based on those criteria, we chose to focus on five peptides with potential template models (Table 1). A F6W or F10W mutation has been introduced into the piscidin and buforin II sequences, respectively, to allow for spectrophotometric quantification and experimental binding measurements leveraging tryptophan fluorescence. All of these structures had a portion of their protein that was both highly homologous to the AMP sequence and bound to DNA in the crystal structure. This final set of peptides should be tractable for our proposed work in an undergraduate research environment while providing us a robust enough set of results to determine whether our design strategy is effective in developing more active AMPs.


![[Pasted image 20240531175326.png]]


## Specific Aim 1: Design and experimentally evaluate peptides for increased DNA binding

In this aim, we will begin by utilizing a computational strategy (Fig. 2) to design arginine mutants of our five chosen AMPs (Table 1) with increased affinity for DNA. We will then assess those predictions experimentally using DNA binding and antimicrobial activity assays. Our preliminary results using this approach to design variants of BF2 are discussed in the final portion of this section.

### Computational Design Process

![[Pasted image 20240531175451.png]]

##### Step 1: 
To begin our process, we will perform MD simulations of each peptide complexed with DNA to generate a conformational ensemble to serve as design “scaffolds.” The creation of initial conformations and parameters for these simulations is based on our previous simulations of BF2 and DesHDAP1 interacting with DNA.[[Grant Proposal And Notes#29. Uyterhoeven, E. T.; Butler, C. H.; Ko, D.; Elmore, D. E., Investigating the nucleic acid interactions and antimicrobial mechanism of buforin II. FEBS Lett. 2008, 582 (12), 1715-1718.| 29]], [[Grant Proposal And Notes#31. Sim, S.; Wang, P.; Beyer, B. N.; Cutrona, K. J.; Radhakrishnan, M. L.; Elmore, D. E., Investigating the nucleic acid interactions of histone-derived antimicrobial peptides. FEBS Lett 2017, 591 (5), 706-717.|31]] The starting structure for these peptide-DNA simulations will be based on the template structures identified for each peptide (Table 1), extracting the sequence of the protein in the structure homologous to the AMP and an adjacent section of DNA at least 20 bp long.

In cases where the full AMP sequence does not have homology with a portion of protein from the crystal structure we will add nonaligned residues on the N- or C-terminus as α-helical structures based on the known or most likely secondary structure of these systems.[[Grant Proposal And Notes#36. Pavia, K. E.; Spinella, S. A.; Elmore, D. E., Novel histone-derived antimicrobial peptides use different antimicrobial mechanisms. Biochim Biophys Acta 2012, 1818 (3), 869-76.| 36]],[[Grant Proposal And Notes#38. Kobayashi, S.; Takeshima, K.; Park, C. B.; Kim, S. C.; Matsuzaki, K., Interactions of the novel antimicrobial peptide buforin 2 with lipid bilayers proline as a translocation promoting factor. Biochemistry 2000, 39 (29), 8648-54.| 38]],[[Grant Proposal And Notes#39. Park, C. B.; Yi, K. S.; Matsuzaki, K.; Kim, M. S.; Kim, S. C., Structure-activity analysis of buforin II, a histone H2A-derived antimicrobial peptide the proline hinge is responsible for the cell-penetrating ability of buforin II. Proc. Natl. Acad. Sci. 2000, 97 (15), 8245-50.| 39]] If necessary, we will add additional base pairs to the extracted structure in the B-DNA conformation. Standard sidechain ionization states at pH 7 will be used in simulations. The histidine residues of piscidin 1 and 3 will be kept in their neutral states based on experimental pKa measurements,[[Grant Proposal And Notes#43. Mihailescu, M.; Sorci, M.; Seckute, J.; Silin, V. I.; Hammer, J.; Perrin, B. S., Jr.; Hernandez, J. I.; Smajic, N.; Shrestha, A.; Bogardus, K. A.; Greenwood, A. I.; Fu, R.; Blazyk, J.; Pastor, R. W.; Nicholson, L. K.; Belfort, G.; Cotten, M. L., Structure and Function in Antimicrobial Piscidins Histidine Position, Directionality of Membrane Insertion, and pH-Dependent Permeabilization. J Am Chem Soc 2019, 141 (25), 9837-9853.| 43]] and we will consider alternate protonation states of the single histidine residues in BF2 and DesHDAP1.

For each AMP-DNA starting structure we will perform three 250 ns replicate MD simulations in explicit solvent using the Amber03 forcefield.[[Grant Proposal And Notes#44. Duan, Y.; Wu, C.; Chowdhury, S.; Lee, M. C.; Xiong, G.; Zhang, W.; Yang, R.; Cieplak, P.; Luo, R.; Lee, T.; Caldwell, J.; Wang, J.; Kollman, P., A point-charge force field for molecular mechanics simulations of proteins based on condensed-phase quantum mechanical calculations. J Comput Chem 2003, 24 (16), 1999-2012.| 44]] The forcefield was chosen because of its robust parameterization for nucleic acid structures and its use in our previous AMP-DNA simulations, and the timescale employed should allow for sufficient conformational flexibility to develop based on our past simulations.[[Grant Proposal And Notes#31. Sim, S.; Wang, P.; Beyer, B. N.; Cutrona, K. J.; Radhakrishnan, M. L.; Elmore, D. E., Investigating the nucleic acid interactions of histone-derived antimicrobial peptides. FEBS Lett 2017, 591 (5), 706-717.| 31]] If it is found that three initial simulations yield too large a standard error (e.g., >5 kcal/mol) in the peptide-DNA electrostatic binding free energy using continuum electrostatic calculations (described below) to robustly compare with mutants, we will perform additional (e.g., 2) MD simulations to increase sampling of the system. The Gromacs simulation package will used for all simulations.[[Grant Proposal And Notes#45. Abraham, M. J.; Murtola, T.; Schulz, R.; Páll, S.; Smith, J. C.; Hess, B.; Lindahl, E., GROMACS High performance molecular simulations through multi-level parallelism from laptops to supercomputers. SoftwareX 2015, 1-2, 19-25.| 45]] Table 2 details rough system sizes, force fields, and simulation times for initial peptide/DNA simulations.

![[Pasted image 20240531175544.png]]
Table 2: General information for proposed simulations of AMPs with DNA or membrane

##### Step 2: 
In our next phase, charge optimization will be carried out on snapshots extracted from the equilibrated portion of each replicate MD simulation performed in Step 1 to coarsely but efficiently identify peptide side chain positions with the potential to increase binding affinity the most by bearing a +1 charge (to mimic arginine substitution). At least 15 snapshots per replicate will be used for the input ensemble. In preliminary work described below, that amount of sampling yielded both computational efficiency and reasonably low statistical variation for initial predictions.

![[Pasted image 20240531175713.png]]

Here, 𝑞<sub>p</sub> and 𝑞<sub>t</sub> are vectors containing the partial atomic charges on the peptide and its target respectively, and 𝑃, 𝑇, and 𝐶 are unit potential matrices for the peptide desolvation, target desolvation, and the complex solvent-screened interaction, respectively. Matrix elements and overall terms in Eq. 1 can be computed by numerically solving the [linearized Poisson-Boltzmann Equation](https://en.wikipedia.org/wiki/Poisson%E2%80%93Boltzmann_equation) (LPBE).[[Grant Proposal And Notes#48. Radhakrishnan, M. L., Designing electrostatic interactions in biological systems via charge optimization or combinatorial approaches insights and challenges with a continuum electrostatic framework. Theor. Chem. Acc. 2012, 131 (8).| 48]]

The electrostatic binding free energy in Equation 1 is a quadratic, convex function of peptide charges and can be minimized by differentiation with respect to 𝑞<sub>p</sub> and setting the resulting expression to zero, yielding the optimal peptide charge distribution for maximal affinity:


![[Pasted image 20240531175755.png]]


While 𝑞<sub>p,opt</sub> is a hypothetical charge distribution, it provides a useful starting point for practical modifications to improve affinity. Constraints can be added in the optimization to ensure physically reasonable charge magnitudes and/or an overall integral charge. Charge optimization has been successfully used by the PI and/or others both to analyze and understand protein systems[[Grant Proposal And Notes#49. Lee, L. P.; Tidor, B., Optimization of binding electrostatics Charge complementarity in the barnase-barstar protein complex. Protein Science 2001, 10 (2), 362-377.| 49]],[[Grant Proposal And Notes#50. Lee, L. P.; Tidor, B., Barstar is electrostatically optimized for tight binding to barnase. Nature Structural Biology 2001, 8 (1), 73-76.| 50]] or drug/small molecule-target systems,[[Grant Proposal And Notes#51. Ahn, J. S.; Radhakrishnan, M. L.; Mapelli, M.; Choi, S.; Tidor, B.; Cuny, G. D.; Musacchio, A.; Yeh, L. A.; Kosik, K. S., Defining Cdk5 ligand chemical space with small molecule inhibitors of Tau phosphorylation. Chemistry & Biology 2005, 12 (7), 811-823.| 51]]-[[Grant Proposal And Notes#54. Sims, P. A.; Wong, C. F.; McCammon, J. A., Charge optimization of the interface between protein kinases and their ligands. J. Comput. Chem. 2004, 25 (11), 1416-1429.|54]] and to successfully design experimentally-validated systems with tighter binding.[[Grant Proposal And Notes#55. Kangas, E.; Tidor, B., Electrostatic complementarity at ligand binding sites Application to chorismate mutase. J. Phys. Chem. B 2001, 105 (4), 880-888.| 55]], [[Grant Proposal And Notes#56. Altman, M. D.; Nalivaika, E. A.; Prabu-Jeyabalan, M.; Schiffer, C. A.; Tidor, B., Computational design and experimental study of tighter binding peptides to an inactivated mutant of HIV-1 protease. Proteins-Structure Function and Bioinformatics 2008, 70 (3), 678-694.|56 ]]

In this work, we will use partial, constrained charge optimization,[[Grant Proposal And Notes#47. Kangas, E.; Tidor, B., Optimizing electrostatic affinity in ligand-receptor binding Theory, computation, and ligand properties. Journal of Chemical Physics 1998, 109 (17), 7522-7545.| 54]],[[Grant Proposal And Notes#57. Li, Q. Progress Toward the Computational Design of Tighter-Binding Antimicrobial Peptides Based on Buforin II. Wellesley College, 2020.| 57]], [[Grant Proposal And Notes#58. Liu, A. Computational Design of Buforin-II-based Antimicrobial Peptides with Improved Electrostatic Interactions with the Cell Membrane. Wellesley College, 2022.|58]] in which portions of the peptide will be optimized while holding other portions constant, via a decomposition of Equation 1 into submatrices and subvectors corresponding to fixed and variable portions of the peptide. To quickly identify peptide positions to mutate to arginine, each existing neutral side chain of an extracted snapshot will be optimized to find its maximal possible affinity were it to assume a +1 charge (mimicking an arginine). As a control, we subtract from this result the maximal possible affinity were it to remain neutral. The resulting ∆∆𝐺 is an estimate of the “affinity gain” that could be achieved from a unit increase in monopole at this position. This estimate is very rough because in addition to assuming any hypothetical atom-centered charge distribution, it assumes the residue retains its shape upon arginine substitution, which is of course untrue. Nevertheless, this approximate analysis acts as a filter for more costly explicit mutant MD simulations in Step 3. We will initially focus on increasing the monopole at one position (i.e., a single arginine mutant) and if multiple such mutants are found to increase affinity, we will consider two positions simultaneously (i.e., increasing the monopole by +2, mimicking double arginine mutants). Once matrix elements for each peptide residue are found via the above method, constrained optimizations are trivially fast, so any desired overall monopole can be efficiently considered, and this method could therefore provide insight into the optimal overall monopole for an AMP of a given length to bind to either a DNA or membrane target.

We will use the DelPhi finite-difference solver[[Grant Proposal And Notes#59. Li, L.; Li, C. A.; Sarkar, S.; Zhang, J.; Witham, S.; Zhang, Z.; Wang, L.; Smith, N.; Petukh, M.; Alexov, E., DelPhi a comprehensive suite for DelPhi software and associated resources. BMC Biophys. 2012, 5, 9.| 59]] and a single-grid red-black successive over-relaxation finite difference solver[[Grant Proposal And Notes#60. Altman, M. D. Computational ligand design and analysis in protein complexes using inverse methods, combinatorial search, and accurate solvation modeling. Ph.D. Thesis, Massachusetts Institute of Technology, Cambridge, 2006.| 60]] to carry out LPBE calculations used in charge optimization. The CONOPT solver[[Grant Proposal And Notes#61. Drud, A., CONOPT - A GRG code for large sparse dynamic nonlinear optimization problems. Mathematical Programming 1985, 31 (2), 153-191.| 61]] within the GAMS software package[[Grant Proposal And Notes#62. Corporation, G. D., General Algebraic Modeling System (GAMS). Fairfax, VA, USA, 2021.| 62]] will be used for constrained, convex optimizations. In this highly charged system, nonlinear ionic strength effects have been shown to affect binding energetics in our previous work[[Grant Proposal And Notes#30. Perez, C. P.; Elmore, D. E.; Radhakrishnan, M. L., Computationally Modeling Electrostatic Binding Energetics in a Crowded, Dynamic Environment Physical Insights from a Peptide-DNA System. J. Phys. Chem. B 2019, 123 (50), 10718-10734.| 30]], but those effects appeared systematic enough that they need not be considered for a design application in which a coarse, more qualitative result is sufficient. Nevertheless, we can again consider nonlinear effects within DelPhi to validate results if needed.

##### Step 3: 
After charge optimization we will have identified 4-5 promising arginine single or multiple mutants with a range of improved binding affinities relative to WT. A robust range of predicted binding affinities will be important to establish relationships between target binding and antimicrobial potency, validate our design strategy, and enable our testing the hypothesis that target binding affinity is sequence-specific and not simply dependent on overall peptide monopole. We will carry out MD simulations of each candidate mutant peptide complexed with DNA. These simulations will allow us to

refine the peptide-DNA structures to account for the conformational changes that arise from the arginine mutations. These MD simulations will be carried out in a similar manner to those in Step 1. As in the first step, we will begin with three replicate simulations, adding up to two additional simulations if needed for statistically significant comparisons of energy calculations in the next step.

##### Step 4: 
The average electrostatic binding energetics of mutants will be estimated by carrying out LPBE calculations on extracted snapshots from equilibrated portions of the MD simulations from Step 3 using the same numerical solvers discussed in Step 2. Since these calculations will be based on the structural ensembles resulting from the MD simulations performed on the mutant structures, they should provide a more refined prediction of the relative DNA binding affinity for the mutations that were originally predicted from charge optimization. At least 3 variants of each AMP with a range of improved binding affinities (e.g., at least 1-2 with highly increased predicted affinity and one with a modest increase to serve as a control) will be chosen for experimental testing.

### Experimental assessment of predictions

Following the computational design approach, we will experimentally assess whether designed variants actually have increased DNA affinity and/or increased antibacterial activity using peptides synthesized with solid-phase synthesis at >95% purity by commercial sources. The relative DNA binding affinity of peptides will be measured using a fluorescence intercalator displacement (FID) assay. We have successfully used this approach to consider DNA affinity of AMPs, including BF2 and DesHDAP1.[[Grant Proposal And Notes#29. Uyterhoeven, E. T.; Butler, C. H.; Ko, D.; Elmore, D. E., Investigating the nucleic acid interactions and antimicrobial mechanism of buforin II. FEBS Lett. 2008, 582 (12), 1715-1718.| 29]],[[Grant Proposal And Notes#31. Sim, S.; Wang, P.; Beyer, B. N.; Cutrona, K. J.; Radhakrishnan, M. L.; Elmore, D. E., Investigating the nucleic acid interactions of histone-derived antimicrobial peptides. FEBS Lett 2017, 591 (5), 706-717.| 31]], [[Grant Proposal And Notes#40 |40]] In the FID assay, DNA is initially mixed with thiazole orange (TO), an intercalator that has an increased fluorescence when bound to DNA.[[Grant Proposal And Notes#63. Nygren, J.; Svanvik, N.; Kubista, M., The interactions between the fluorescent dye thiazole orange and DNA. Biopolymers 1998, 46, 39-51.| 63]] The DNA•TO complex is then titrated with peptide, leading to decreased fluorescence. Peptide variants can be compared by the concentration of peptide necessary to reduce TO fluorescence by 50% (C50), although the resulting titration data also can be used to calculate binding constants of peptide•DNA interactions. dsDNA for these experiments will be obtained from a commercial source for DNA oligonucleotide synthesis. For each peptide, a dsDNA molecule matching the sequence used in the modeling will be used to allow for direct comparison to computations. In previous work we showed that wild type and mutant versions of BF2 and DesHDAP1 do not show sequence specific binding of DNA.[[Grant Proposal And Notes#31. Sim, S.; Wang, P.; Beyer, B. N.; Cutrona, K. J.; Radhakrishnan, M. L.; Elmore, D. E., Investigating the nucleic acid interactions of histone-derived antimicrobial peptides. FEBS Lett 2017, 591 (5), 706-717.| 31]] We will verify whether this is true for piscidin and lasioglossin by also comparing their affinity for polyA•T, polyG•C and a mixed A•T/G•C dsDNA.

The antibacterial activity of the designed peptides will be measured against standard laboratory strains of E. coli and B. subtilis using both radial diffusion (RDA) and microbroth dilution assays[[Grant Proposal And Notes#64. Lehrer, R. I.; Rosenman, M.; Harwig, S. S. S. L.; Jackson, R.; Eisenhauer, P., Ultrasensitive assays for endogenous antimicrobial polypeptides. J. Immunol. Methods 1991, 137, 167-173.| 64]],[[Grant Proposal And Notes#65. Steinberg, D. A.; Lehrer, R. I., Designer assays for antimicrobial peptides. Disputing the "one-size-fits-all" theory. Methods. Mol. Biol. 1997, 78, 169-186.| 65]]. In RDA experiments, bacteria are incubated inside small holes in an agar plate. Each hole contains a set concentration of a peptide or a buffer control, and the diameter of bacterial clearance around these holes is measured after an overnight incubation. Because the peptides form a concentration gradient as they diffuse out of the hole into the plate, a larger region of clearance implies a more active peptide. Alternatively, microbroth dilution assays will expose bacteria in liquid suspensions to different concentrations of AMPs in order to determine the minimum inhibitory concentration (MIC) of peptides. Using both methods will help ensure robust comparisons of activity.

In addition to providing an assessment of our design approach, these experiments will also allow us to determine whether there is a correlation between DNA binding affinity and antibacterial activity for each of the peptides. In previous work, the co-investigator used this relationship to help confirm the role of DNA binding in the proposed mechanism of BF2.[[Grant Proposal And Notes#29. Uyterhoeven, E. T.; Butler, C. H.; Ko, D.; Elmore, D. E., Investigating the nucleic acid interactions and antimicrobial mechanism of buforin II. FEBS Lett. 2008, 582 (12), 1715-1718.| 29]] Although there are confounding factors that would prevent a perfect correlation between these characteristics, such as alterations in membrane interactions, one would expect to observe a clear relationship between DNA binding and activity if DNA binding is a bone fide part of the mechanism of action.
 
### Preliminary results: BF2 design and experimental assessment

We have begun a preliminary application of our proposed design process with BF2 – here we briefly describe this initial implementation of the five steps above:
  
##### Step 1: 
Three replicate 250 ns simulations of BF2 complexed with DNA were carried out as part of prior work[[Grant Proposal And Notes#30. Perez, C. P.; Elmore, D. E.; Radhakrishnan, M. L., Computationally Modeling Electrostatic Binding Energetics in a Crowded, Dynamic Environment Physical Insights from a Peptide-DNA System. J. Phys. Chem. B 2019, 123 (50), 10718-10734.| 30]] using specifications shown in Table 2. Fig. 4 shows a sample snapshot of the complex (with water and ions omitted for clarity).

![[Pasted image 20240531180223.png]]


##### Step 2: 
15 snapshots, taken 3.5 ns apart from the last 50 ns of each replicate, were extracted for a total of 45 snapshots. For each snapshot, charge optimizations were performed for each neutral side chain position in turn, excluding W10 and P11 due to the roles these amino acids are known to play in experimental quantification and translocation,[[Grant Proposal And Notes#39. Park, C. B.; Yi, K. S.; Matsuzaki, K.; Kim, M. S.; Kim, S. C., Structure-activity analysis of buforin II, a histone H2A-derived antimicrobial peptide the proline hinge is responsible for the cell-penetrating ability of buforin II. Proc. Natl. Acad. Sci. 2000, 97 (15), 8245-50.| 39]],[[Grant Proposal And Notes#40. Xie, Y.; Fleming, E.; Chen, J. L.; Elmore, D. E., Effect of proline position on the antimicrobial mechanism of buforin II. Peptides 2011, 32 (4), 677-82.| 40]] respectively. Fig. 5 displays the average ∆∆𝐺 , or difference in optimal electrostatic binding free energy, when a side chain has a +1 monopole vs. when it is neutral (error bars show standard error over 3 replicates). The positions with most negative ∆∆𝐺 are predicted to increase affinity the most upon becoming positively charged, assuming optimal interactions. Based on this coarse initial model, position T1R was chosen as a promising mutant to enhance DNA binding, while A6R, L8R, Q9R, and L19R were among the potential substitutions predicted to have a more minimal effect. Notably, the level of binding enhancement was predicted to be position-dependent and not solely related to increased overall peptide charge.

##### Steps 3 and 4: 
In these initial runs, replicate simulations were performed for the T1R, A6F, L8R, and Q9R mutant peptides complexed with DNA (L19R was initially considered but after one simulation its affinity appeared too similar to that of T1R and it was not considered further), and electrostatic binding free energies were computed on 51 snapshots over the last 25 ns of each replicate using a finite-difference LBPE solver60 using a grid resolution of 3.7 grids/Å at the highest focusing, inner/outer dielectric constants of 4 and 80 respectively, and an ionic strength of 0.1M to match the MD simulation conditions. Table 3 summarizes those results. Indeed, these subsequent calculations supported the idea that the T1R variant has higher electrostatic affinity to DNA than the WT peptide. When taking structural flexibility due to explicit arginine substitution into account, L8R and Q9R still were predicted to have more modest impacts on DNA binding, in agreement with charge optimization, but the A6R mutation was now predicted to show a similar enhancement to T1R. These changes highlight the necessity of incorporating these additional steps of structural refinement versus relying solely on a rigid optimization model for design, and they can also serve to continually refine or inform our choice of optimization criteria and constraints in Step 2 if needed as the project moves forward.
We also utilized these simulations to consider the impact of H16 in both its deprotonated and protonated state, as there is no experimental pKa data available on that position. Not shown here are results from additional simulations and analyses conducted to better understand why the protonated histidine differentially affects mutants, including simulations of double mutants where this histidine
![[Pasted image 20240531180320.png]]

was mutated to a phenylalanine to remove the possibility of protonation.[[Grant Proposal And Notes#57. Li, Q. Progress Toward the Computational Design of Tighter-Binding Antimicrobial Peptides Based on Buforin II. Wellesley College, 2020.| 57]] While H16 protonation did not impact the relative order of binding for most variants, it did alter the predicted impact of Q9R compared to other variants. Results from future peptide-DNA binding measurements like those described below will allow us to validate which protonation model most reliably reflects the experimental system.

As a result of the computational workflow, T1R and L8R were chosen as candidates for experimental testing in this preliminary study, as they were predicted to show a range of improved binding affinities relative to WT (T1R > L8R > WT).

##### Step 5: 
We then experimentally compared the DNA binding and antimicrobial activity of the T1R and
L8R variants predicted to have differing levels of enhanced DNA binding. As predicted, both mutations enhanced DNA binding (a lower C50), with T1R showing a greater
enhancement (Fig. 6), consistent with our computations (Table 3). Similarly, T1R and L8R showed a
significant increase in activity against E. coli in radial diffusion assays (Fig. 7). T1R also showed increased activity in initial microbroth dilution assays (data not shown). Thus, our approach appeared to be successful in our goals of designing BF2 variants with increased DNA binding and antimicrobial activity.

![[Pasted image 20240531180639.png]]

![[Pasted image 20240531181214.png]]

![[Pasted image 20240531181236.png]]

We note that our computational models focused on electrostatic interactions and, thus far, such a model has proved sufficient for reasonable qualitative prediction. Nevertheless, we have also carried out preliminary calculations to assess the experimentally- tested mutations’ effects on the nonpolar components of binding free energy (e.g., van der Waals modeled by a Lennard-Jones potential and solvent entropy via a solvent-accessible surface area term, implemented in Gromacs).

Interestingly, including these components led to results that were not consistent with our experimental observations, supporting the idea that using a simple model incorporating electrostatics may be not only sufficient but might in fact be more effective for design predictions in this highly charged system. Nevertheless, we will continue to assess these nonpolar components in order to address the important questions of what computational design model in this system best balances efficiency and accuracy and whether electrostatics is in fact optimal for predictions more generally in these systems.

## Specific Aim 2: Design and experimentally evaluate peptides for increased membrane interactions

In this aim, we will adapt the computational design strategy outlined in Specific Aim 1 to design variants of our five AMPs (Table 1) that have increased affinity for the lipid membrane. We will then assess those predictions experimentally using lipid membrane affinity and antimicrobial activity assays. In this case, we hypothesize that a moderate increase of membrane affinity will enhance activity. However, we also hypothesize that this may be a limited effect as increasing affinity too much may lead to a decreased ability of peptides to translocate across membranes as they become tightly membrane associated. Our design and verification approach will allow us to address this hypothesis. Our preliminary results using this design approach for BF2 are discussed at the end of this section.
### Computational design process
For our design of peptide variants with increased membrane interactions, we will follow an analogous four-step computational process to that outlined in Specific Aim 1 (Fig. 2). However, due to the differences in DNA versus membrane systems we will need to make the following modifications to the initial structure generation and details for simulations.

Initial structures will consist of four peptide molecules parallel to the surface of phospholipid bilayer with a 3:1 POPE:POPG composition prepared using CHARMM-GUI66 followed by a brief 10 ns MD relaxation to reduce unfavorable interactions. Peptides will be started in a fully a-helical conformation as the peptides used in this study are primarily a-helical when bound to membranes, and we found that regions of BF2 known to be more disordered rapidly lost helical conformations in past simulations.[[Grant Proposal And Notes#67. Elmore, D. E., Insights into buforin II membrane translocation from molecular dynamics simulations. Peptides 2012, 38 (2), 357-62.| 67]] Four peptides were chosen both because of the hypothesis that peptides may interact to facilitate membrane translocation [[Grant Proposal And Notes#37. Kobayashi, S.; Chikushi, A.; Tougu, S.; Imura, Y.; Nishida, M.; Yano, Y.; Matsuzaki, K., Membrane translocation mechanism of the antimicrobial peptide buforin 2. Biochemistry 2004, 43, 15610-15616.|37]] and to provide more peptide conformational data if needed per replicate, given the longer equilibration times needed per membrane replicate vs. DNA replicate. Each replicate simulation will be carried out for 1 μs to enable the membrane adequate time to equilibrate and will use the CHARMM36 force field, which is parametrized for lipids and proteins and was used in our previous AMP-membrane simulations.[[Grant Proposal And Notes#67. Elmore, D. E., Insights into buforin II membrane translocation from molecular dynamics simulations. Peptides 2012, 38 (2), 357-62.| 67]] See Table 2 above for details. Charge optimizations of peptide residues and electrostatic binding free energy calculations on extracted snapshots in Steps 2 and 4 will proceed similarly to those described in Aim 1. Notably, this approach will allow us to select peptides predicted to have differing levels of enhanced membrane affinity.

### Experimental assessment of predictions

First, we will use a combination of antimicrobial assays and lipid vesicle measurements to experimentally evaluate the relationship between AMP activity and membrane affinity. Antimicrobial assays will be performed using the microbroth dilution and radial diffusion assays described in Aim 1. Membrane affinity assays will be performed using lipid vesicles with a 3:1 POPE:POPG composition to mirror our simulation conditions and reflect a lipid composition analogous to that of bacterial cells. In this experiment, we will titrate peptides with vesicles. As peptides associate with the membrane, their emission spectra experience a blue shift to shorter wavelengths, and one can use the fluorescence change at a given wavelength—not the wavelength of the maximum emission— to determine membrane affinity.[[Grant Proposal And Notes#68. Ladokhin, A. S.; Jayasinghe, S.; White, S. H., How to measure and analyze tryptophan fluorescence in membrane properly, and why bother? Anal Biochem 2000, 285 (2), 235-245.| 68]]-[[Grant Proposal And Notes#72. Ladokhin, A. S.; Jayasinghe, S.; White, S. H., How to measure and analyze tryptophane fluorescence in membrane properly, and why bother? Anal Biochem 2000, 285 (2), 235-245.|72]] The co-investigator’s lab has successfully used this approach, incorporating correction factors for peptide•cuvette adsorption and light scattering of vesicles, to consider AMP•membrane interactions in previous studies.[[Grant Proposal And Notes#42. Cutrona, K. J.; Kaufman, B. A.; Figueroa, D. M.; Elmore, D. E., Role of arginine and lysine in the antimicrobial mechanism of histone-derived antimicrobial peptides. FEBS Lett 2015, 589 (24 Pt B), 3915-20.| 42]],[[Grant Proposal And Notes#73. Fleming, E.; Maharaj, N. P.; Chen, J. L.; Nelson, R. B.; Elmore, D. E., Effect of lipid composition on buforin II structure and membrane entry. Proteins 2008, 73 (2), 480-491.| 73]]

We will then use additional lipid vesicle assays to more fully understand the relationship between membrane affinity and the ability of peptides to permeabilize or translocate across membranes. To measure permeabilization we will utilize a dye-leakage assay in which we encapsulate a florescence dye (ANTS) and a quencher (DPX) into vesicles.[[Grant Proposal And Notes#37. Kobayashi, S.; Chikushi, A.; Tougu, S.; Imura, Y.; Nishida, M.; Yano, Y.; Matsuzaki, K., Membrane translocation mechanism of the antimicrobial peptide buforin 2. Biochemistry 2004, 43, 15610-15616.| 37]],[[Grant Proposal And Notes#38. Kobayashi, S.; Takeshima, K.; Park, C. B.; Kim, S. C.; Matsuzaki, K., Interactions of the novel antimicrobial peptide buforin 2 with lipid bilayers proline as a translocation promoting factor. Biochemistry 2000, 39 (29), 8648-54.| 38]],[[Grant Proposal And Notes#74. Epand, R. M.; Epand, R. F., Liposomes as models for antimicrobial peptides. Methods Enzymol 2003, 372, 124-133.| 74]] The amount of membrane permeabilization can be quantified by the increase in fluorescence when these vesicles are exposed to an AMP variant because disruption of the membrane will separate the fluorescent ANTS from the DPX quencher. For translocation we will utilize an assay originally implemented by Matsuzaki and co-workers and further refined in the co-investigator’s laboratory to compare translocation of AMP variants.[[Grant Proposal And Notes#37. Kobayashi, S.; Chikushi, A.; Tougu, S.; Imura, Y.; Nishida, M.; Yano, Y.; Matsuzaki, K., Membrane translocation mechanism of the antimicrobial peptide buforin 2. Biochemistry 2004, 43, 15610-15616.| 37]],[[Grant Proposal And Notes#38. Kobayashi, S.; Takeshima, K.; Park, C. B.; Kim, S. C.; Matsuzaki, K., Interactions of the novel antimicrobial peptide buforin 2 with lipid bilayers proline as a translocation promoting factor. Biochemistry 2000, 39 (29), 8648-54.| 38]],[[Grant Proposal And Notes#75. Spinella, S. A.; Nelson, R. B.; Elmore, D. E., Measuring peptide translocation into large unilamellar vesicles. J Vis Exp 2012, (59), e3571.| 75]] In this assay, we encapsulate trypsin into vesicles doped with fluorescently labeled lipids. Peptide that remains affiliated with the membrane on the outside the vesicle will exhibit a FRET response with the vesicles. However, any peptide that translocates into the vesicle will be degraded, leading to a loss of fluorescence. Results from these two experiments will help provide an explanation for the relationships observed between membrane affinity and antimicrobial activity for designed variants.

### Preliminary results from design and experimental assessment with BF2 

We have begun Steps 1 and 2 in the computational design process toward this aim. Specifically, toward Step 1, we have thus far carried out 2 replicate 1 μs simulations of BF2/membrane systems and 2 replicate 1 μs simulations of Des1HDAP/membrane systems. [[Grant Proposal And Notes#76. Huang, J. Investigating the Effects of Crowding on Antimicrobial Peptides Using Molecular Dynamics Simulations. Wellesley College, 2021.|76]] (replicate 1, 1000 ns) is shown in Fig. 8 (note that periodic boundary conditions are employed and molecules were extracted to remain whole and therefore not necessarily within the simulation box).

![[Pasted image 20240531182029.png]]

Toward Step 2, we have carried out charge optimization calculations on 25 BF2/membrane snapshots sampled at 10 ns intervals from the last 250 ns of one replicate thus far, choosing one of the four peptides to optimize (in the absence of the other three peptides). The results are presented in Fig. 9. While the results are only from one replicate, it is interesting to note that position 1, which was predicted to improve binding the most to DNA upon bearing a +1 charge, is one of the least improved with our current sampling in this system, suggesting that different variants might optimize peptide affinity toward DNA vs. lipid, a prediction that ultimately would be experimentally tested if it is robustly observed. Based on these preliminary results, V15R appears to be a promising candidate, and analyses on further replicates is ongoing. The large standard deviation in the predicted change in optimal binding free energy for G13 may be due to the very small number of atoms being optimized (2), perhaps making it more sensitive to conformational dynamics and membrane interactions; a deeper analysis of these results, which will also analyze the role conformational flexibility around this residue, is ongoing.

![[Pasted image 20240531182304.png]]

### Refinement of computational models used to predict DNA and membrane affinity
In both Aims 1 and 2, we will continue to refine our computational models when there is qualitative discrepancy between predictions in Step 4 of our workflow and the experimental outcomes. This may involve carrying out additional replicates to increase sampling, incorporating nonpolar components or nonlinear electrostatic models into our energy functions, or re-evaluating our treatment of titratable residues. As an additional way to provide more holistic insights into the cause of discrepancies, should they occur, the PI may also incorporate combinatorial design strategies, which simulate structures of mutated/designed complexes by optimally repacking side chains off re-sample backbone conformations and can therefore provide a “middle” ground between more approximate rigid charge optimization models and costly explicit MD simulations. In previous work, the PI has applied earlier versions of current combinatorial design methods toward both protein[[Grant Proposal And Notes#77. Zhang, Y. X. L.; Radhakrishnan, M. L.; Lu, X. H.; Gross, A. W.; Tidor, B.; Lodish, H. F., Symmetric Signaling by an Asymmetric 1 Erythropoietin 2 Erythropoietin Receptor Complex. Mol. Cell 2009, 33 (2), 266-274.| 77]] and A sample snapshot from a BF2/membrane

small-molecule[[Grant Proposal And Notes#78. Shen, Y.; Radhakrishnan, M. L.; Tidor, B., Molecular mechanisms and design principles for promiscuous inhibitors to avoid drug resistance Lessons learned from HIV-1 protease inhibition. Proteins-Structure Function and Bioinformatics 2015, 83 (2), 351-372.| 78]] design and has also reviewed aspects of them as part of a larger review on electrostatic design.[[Grant Proposal And Notes#48. Radhakrishnan, M. L., Designing electrostatic interactions in biological systems via charge optimization or combinatorial approaches insights and challenges with a continuum electrostatic framework. Theor. Chem. Acc. 2012, 131 (8).| 48]] As part of potential parallel work to the proposed project, she is initiating a collaboration with Bruce Donald, the lead developer of the Osprey combinatorial design software package,[[Grant Proposal And Notes#79. Hallen, M. A.; Martin, J. W.; Ojewole, A.; Jou, J. D.; Lowegard, A. U.; Frenkel, M. S.; Gainza, P.; Nisonoff, H. M.; Mukund, A.; Wang, S.; Holt, G. T.; Zhou, D.; Dowd, E.; Donald, B. R., OSPREY 3.0 Open-source protein redesign for you, with powerful new features. J Comput Chem 2018, 39 (30), 2494-2507.| 79]] to more broadly extend and apply protein design methodologies within Osprey to include non-protein binding targets such as those in this work. While that project is distinct from the design approach in this proposal, it is possible that insights from that work could be integrated into this project to provide an additional tool to aid in the refinement of the computational models in the proposed workflow.

## Specific Aim 3: Design peptides optimized for both DNA binding and membrane interactions

Once the relationships between (i) DNA binding affinity and antimicrobial potency and (ii) membrane binding affinity and potency have been experimentally determined for a given peptide, we will work toward designing peptides that account simultaneously for both DNA and membrane targets to maximize potency in a more holistic manner.

As a preliminary step, we will computationally and experimentally evaluate the binding affinity of the peptides designed toward a single target in each of Aims 1 and 2 (i.e., variants based only on DNA- binding and variants based only on membrane-binding) toward the target for which it was not designed. Doing so will help us understand whether increased affinity for both targets may already occur as a byproduct of optimizing toward only one target. Our next step toward this aim will be to use the “naïve” strategy of simply combining the mutations from the most potent single-target designs found in Aims 1 and 2. We will carry out MD simulations of these variants, computationally analyze their binding interactions and affinity, and experimentally evaluate both their antimicrobial potency and affinity for DNA and membranes using the procedures outlined in Aims 1 and 2.

As a more rational and potentially effective approach at determining potent translocating AMPs, we will carry out designs where we explicitly consider both targets – membrane and DNA – simultaneously. In order to carry out this multi-target design, we first use the data from the most refined computational model and workflow obtained from Aims 1 and 2 to define thresholds or ranges for predicted binding affinities for each target (DNA or membrane) that experimentally yield high potency (e.g., within the top third of design candidates whose experimental potency validated computational predictions). A threshold will be used if affinity and potency correlate for a target, whereas a range (or ranges) will be used if other, non-monotonic relationships were found.

The multitarget optimizations carried out will each be formulated with multiple objective functions, with a goal of comparing their predictions with each other and with experimental measurements to help provide robust multi-target design strategies in future applications. In the first formulation, the sum of the electrostatic binding free energies to both targets (averaged over all considered snapshots) will be minimized subject to the constraint that the binding affinity improvement toward each target is either better than a particular threshold or within a certain range. A similar approach has been previously used by the PI to optimize electrostatic properties of the HIV-1 reverse transcriptase inhibitor rilpivirine toward three clinically-relevant variants of HIV-1 reverse transcriptase, [[Grant Proposal And Notes#53. Minkara, M. S.; Davis, P. H.; Radhakrishnan, M. L., Multiple drugs and multiple targets An analysis of the electrostatic determinants of binding between non-nucleoside HIV-1 reverse transcriptase inhibitors and variants of HIV-1 RT. Proteins-Structure Function and Bioinformatics 2012, 80, 573-590.|53]] and in that study, it was predicted that a more hydrophobic drug would more promiscuously target all variants. As in Aim 1, each neutral residue (and pairs of neutral residues) will be optimized assuming side chain monopoles constrained to be either +1 (mimicking an arginine residue) or 0, and the differences between these optimal energetics will be compared across possible designs to identify positions for arginine substitution. This formulation provides a convex optimization problem that, if feasible, can be solved to global optimality. Follow-up MD simulations of ~3 promising variants will then be carried out and analyzed prior to experimental testing.

The second formulation will determine a charge distribution for a given set of side chain monopoles that maximizes, beyond a minimal threshold, the number of snapshots “covered” in each target ensemble, where “coverage” is defined by having affinities below appropriate thresholds or within a specified range for each target. This optimization can be used to complement the first formulation, especially because it may yield feasible solutions in cases in which the first formulation does not. It is an integer-programming-based method related to those developed by the PI that were previously applied to both theoretical model systems and HIV-protease combinatorial drug design[[Grant Proposal And Notes#78. Shen, Y.; Radhakrishnan, M. L.; Tidor, B., Molecular mechanisms and design principles for promiscuous inhibitors to avoid drug resistance Lessons learned from HIV-1 protease inhibition. Proteins-Structure Function and Bioinformatics 2015, 83 (2), 351-372.| 78]],[[Grant Proposal And Notes#80. Radhakrishnan, M. L.; Tidor, B., Optimal drug cocktail design Methods for targeting molecular ensembles and insights from theoretical model systems. J. Chem Inf. Model. 2008, 48 (5), 1055-1073.| 80]]. For both sets of formulations, the GAMS software package, and specifically the CONOPT, CPLEX, and/or BARON solvers will be used for optimizations. Again, we will follow-up charge optimization-based designs with MD simulations of explicit variants as well as experimental measurements of their antimicrobial activity, DNA binding and membrane affinity.

## Timeline and appropriateness of project for undergraduate environment

Aims 1 and 2 can be performed simultaneously. We plan to focus on those aims for an initial two peptides for which we have significant preliminary data, BF2 and DesHDAP1, during the first year of the grant period. Based on those results, we will then be able to begin work on Aim 3 for those two peptides during the second year, while we begin work on the other three peptides through Aims 1 and 2. This work on the subsequent peptides will be more efficient given our efforts to optimize our workflow for BF2 and DesHDAP1. In the third year we will complete the work for the final peptides through the final aim.

All of the preliminary computational and experimental data in this proposal were collected by Wellesley undergraduate students. The project is set up such that individual students can take the lead on designing different peptides. The scope of student projects can be easily tailored to their interest and length of time in the lab. For example, students who are opting for a shorter research experience could focus on MD simulations or electrostatics calculations for computational work or DNA binding or antimicrobial assays for experimental work. However, students who work for a longer- term, including an honors thesis project, could be involved in multiple computational and/or experimental approaches. Even students who are not personally involved with both computational and experimental aspects will be able to better understand connections between those approaches through discussions of their data with students involved with other aspects at lab meetings. The PI and co-investigator have a strong track record of mentoring students on collaborative projects, and many of those students have progressed into top graduate and medical programs (e.g., Harvard, Yale, Stanford and Boston University PhD, Michigan MD/PhD, and Rochester MD).

Importantly, many of the approaches described in this proposal here have proven amenable to students who are just starting research, such as those who may be participating in many of our campus programs targeting early research experiences for students who come from underrepresented backgrounds in the sciences. Both the PI and co-investigator regularly participate in those programs, and both have regularly worked with students in the college’s First Year Apprenticeship Program, Sophomore Early Research Program and/or McNair Scholars Program. During the ’21-22 academic year approximately 40% of students in the Radhakrishnan and Elmore Labs identified with groups underrepresented in STEM.


# Bibliography and References Cited

###### 1. Centers for Disease Control and Prevention. Antibiotic resistance threats in the United States; 2013.
###### 2. Shrestha, P.; Cooper, B. S.; Coast, J.; Oppong, R.; Do Thi Thuy, N.; Phodha, T.; Celhay, O.; Guerin, P. J.; Wertheim, H.; Lubell, Y., Enumerating the economic cost of antimicrobial resistance per antibiotic consumed to inform the evaluation of interventions affecting their use. Antimicrob Resist Infect Control 2018, 7, 98.
###### 3. Thorpe, K. E.; Joski, P.; Johnston, K. J., Antibiotic-Resistant Infection Treatment Costs Have Doubled Since 2002, Now Exceeding $2 Billion Annually. Health Aff (Millwood) 2018, 37 (4), 662-669.
###### 4. World Health Organization. Antibacterial agents in clinical and preclinical development: an overview and analysis; 2021.
###### 5. Nizet, V., Antimicrobial peptide resistance mechanisms of human bacterial pathogens. Curr Issues Mol Biol 2006, 8 (1), 11-26.
###### 6. Spohn, R.; Daruka, L.; Lazar, V.; Martins, A.; Vidovics, F.; Grezal, G.; Mehi, O.; Kintses, B.; Szamel, M.; Jangir, P. K.; Csorgo, B.; Gyorkei, A.; Bodi, Z.; Farago, A.; Bodai, L.; Foldesi, I.; Kata, D.; Maroti, G.; Pap, B.; Wirth, R.; Papp, B.; Pal, C., Integrated evolutionary analysis reveals antimicrobial peptides with limited resistance. Nat Commun 2019, 10 (1), 4538.
###### 7. Zasloff, M., Antimicrobial peptides of multicellular organisms. Nature 2002, 415 (6870), 389-95.
###### 8. Grassi, L.; Maisetta, G.; Esin, S.; Batoni, G., Combination Strategies to Enhance the Efficacy of Antimicrobial Peptides against Bacterial Biofilms. Front Microbiol 2017, 8, 2409.
###### 9. Naghmouchi, K.; Le Lay, C.; Baah, J.; Drider, D., Antibiotic and antimicrobial peptide combinations: synergistic inhibition of Pseudomonas fluorescens and antibiotic-resistant variants. Res Microbiol 2012, 163 (2), 101-8.
###### 10. Pletzer, D.; Mansour, S. C.; Hancock, R. E. W., Synergy between conventional antibiotics and anti-biofilm peptides in a murine, sub-cutaneous abscess model caused by recalcitrant ESKAPE pathogens. PLoS Pathog 2018, 14 (6), e1007084.
###### 11. Yu, G.; Baeder, D. Y.; Regoes, R. R.; Rolff, J., Combination effects of antimicrobial peptides. Antimicrob Agents Chemother 2016, 60 (3), 1717-24.
###### 12. Zharkova, M. S.; Orlov, D. S.; Golubeva, O. Y.; Chakchir, O. B.; Eliseev, I. E.; Grinchuk, T. M.; Shamova, O. V., Application of Antimicrobial Peptides of the Innate Immune System in Combination With Conventional Antibiotics-A Novel Way to Combat Antibiotic Resistance? Front Cell Infect Microbiol 2019, 9, 128.
###### 13. Ghosh, A.; Kar, R. K.; Jana, J.; Saha, A.; Jana, B.; Krishnamoorthy, J.; Kumar, D.; Ghosh, S.; Chatterjee, S.; Bhunia, A., Indolicidin targets duplex DNA: structural and mechanistic insight through a combination of spectroscopy and microscopy. ChemMedChem 2014, 9 (9), 2052-8.
###### 14. Kawasaki, H.; Iwamuro, S., Potential roles of histones in host defense as antimicrobial agents. Infect. Disord. Drug Targets 2008, 8 (3), 195-205.
###### 15. Libardo, M. D. J.; Bahar, A. A.; Ma, B.; Fu, R.; McCormick, L. E.; Zhao, J.; McCallum, S. A.; Nussinov, R.; Ren, D.; Angeles-Boza, A. M.; Cotten, M. L., Nuclease activity gives an edge to host-defense peptide piscidin 3 over piscidin 1, rendering it more effective against persisters and biofilms. FEBS J 2017, 284 (21), 3662-3683.
###### 16. Lu, X.; Shen, J.; Jin, X.; Ma, Y.; Huang, Y.; Mei, H.; Chu, F.; Zhu, J., Bactericidal activity of Musca domestica cecropin (Mdc) on multidrug-resistant clinical isolate of Escherichia coli. Appl Microbiol Biotechnol 2012, 95 (4), 939-45.
###### 17. Wang, G., Tool developments for structure-function studies of host defense peptides. Protein Pept Lett 2007, 14 (1), 57-69.
###### 18. Bandyopadhyay, S.; Lee, M.; Sivaraman, J.; Chatterjee, C., Model membrane interaction and DNA-binding of antimicrobial peptide Lasioglossin II derived from bee venom. Biochem Biophys Res Commun 2013, 430 (1), 1-6.
###### 19. Baumann, A.; Demoulins, T.; Python, S.; Summerfield, A., Porcine cathelicidins efficiently complex and deliver nucleic acids to plasmacytoid dendritic cells and can thereby mediate bacteria-induced IFN-alpha responses. J Immunol 2014, 193 (1), 364-71.
###### 20. Zhang, J.; Wu, X.; Zhang, S. Q., Antifungal mechanism of antibacterial peptide, ABP-CM4, from Bombyx mori against Aspergillus niger. Biotechnol Lett 2008, 30 (12), 2157-63.
###### 21. Sapkota, M.; Adnan Qureshi, M.; Arif Mahmud, S.; Balikosa, Y.; Nguyen, C.; Boll, J. M.; Pellegrino, M. W., A nematode-derived, mitochondrial stress signaling-regulated peptide exhibits broad antibacterial activity. Biol Open 2021, 10 (5).
###### 22. Zhang, K.; Zhang, H.; Gao, C.; Chen, R.; Li, C., Antimicrobial Mechanism of pBD2 against Staphylococcus aureus. Molecules 2020, 25 (15).
###### 23. Hansmann, B.; Schroder, J. M.; Gerstel, U., Skin-Derived C-Terminal Filaggrin-2 Fragments Are Pseudomonas aeruginosa-Directed Antimicrobials Targeting Bacterial Replication. PLoS Pathog 2015, 11 (9), e1005159.
###### 24. Li, Z.; Jing, X.; Yuan, Y.; Shui, Y.; Li, S.; Zhao, Z.; Deng, B.; Zhang, W., In vitro and in vivo Activity of Phibilin Against Candida albicans. Front Microbiol 2022, 13, 862834.
###### 25. Rahman, M. S.; Choi, Y. H.; Choi, Y. S.; Yoo, J. C., Glycin-rich antimicrobial peptide YD1 from B. amyloliquefaciens, induced morphological alteration in and showed affinity for plasmid DNA of E. coli. AMB Express 2017, 7 (1), 8.
###### 26. Yi, T.; Huang, Y.; Chen, Y., Production of an antimicrobial peptide AN5-1 in Escherichia coli and its dual mechanisms against bacteria. Chem Biol Drug Des 2015, 85 (5), 598-607.
###### 27. Park, C. B.; Kim, H. S.; Kim, S. C., Mechanism of action of the antimicrobial peptide buforin II: buforin II kills microorganisms by penetrating the cell membrane and inhibiting cellular functions. Biochem. Biophys. Res. Commun. 1998, 244 (1), 253-7.
###### 28. Tsao, H. S.; Spinella, S. A.; Lee, A. T.; Elmore, D. E., Design of novel histone- derived antimicrobial peptides. Peptides 2009, 30 (12), 2168-73.
###### 29. Uyterhoeven, E. T.; Butler, C. H.; Ko, D.; Elmore, D. E., Investigating the nucleic acid interactions and antimicrobial mechanism of buforin II. FEBS Lett. 2008, 582 (12), 1715-1718.
###### 30. Perez, C. P.; Elmore, D. E.; Radhakrishnan, M. L., Computationally Modeling Electrostatic Binding Energetics in a Crowded, Dynamic Environment: Physical Insights from a Peptide-DNA System. J. Phys. Chem. B 2019, 123 (50), 10718-10734.
###### 31. Sim, S.; Wang, P.; Beyer, B. N.; Cutrona, K. J.; Radhakrishnan, M. L.; Elmore, D. E., Investigating the nucleic acid interactions of histone-derived antimicrobial peptides. FEBS Lett 2017, 591 (5), 706-717.
###### 32. Jenssen, H.; Hamill, P.; Hancock, R. E., Peptide antimicrobial agents. Clin. Microbiol. Rev. 2006, 19 (3), 491-511.
###### 33. Yeaman, M. R.; Yount, N. Y., Mechanisms of antimicrobial peptide action and resistance. Pharmacol. Rev. 2003, 55 (1), 27-55.
###### 34. Hale, J. D.; Hancock, R. E., Alternative mechanisms of action of cationic antimicrobial peptides on bacteria. Expert Rev Anti Infect Ther 2007, 5 (6), 951-9.
###### 35. Wang, G.; Li, X.; Wang, Z., APD3: the antimicrobial peptide database as a tool for research and education. Nucleic Acids Res 2016, 44 (D1), D1087-93.
###### 36. Pavia, K. E.; Spinella, S. A.; Elmore, D. E., Novel histone-derived antimicrobial peptides use different antimicrobial mechanisms. Biochim Biophys Acta 2012, 1818 (3), 869-76.
###### 37. Kobayashi, S.; Chikushi, A.; Tougu, S.; Imura, Y.; Nishida, M.; Yano, Y.; Matsuzaki, K., Membrane translocation mechanism of the antimicrobial peptide buforin 2. Biochemistry 2004, 43, 15610-15616.
###### 38. Kobayashi, S.; Takeshima, K.; Park, C. B.; Kim, S. C.; Matsuzaki, K., Interactions of the novel antimicrobial peptide buforin 2 with lipid bilayers: proline as a translocation promoting factor. Biochemistry 2000, 39 (29), 8648-54.
###### 39. Park, C. B.; Yi, K. S.; Matsuzaki, K.; Kim, M. S.; Kim, S. C., Structure-activity analysis of buforin II, a histone H2A-derived antimicrobial peptide: the proline hinge is responsible for the cell-penetrating ability of buforin II. Proc. Natl. Acad. Sci. 2000, 97 (15), 8245-50.
###### 40. Xie, Y.; Fleming, E.; Chen, J. L.; Elmore, D. E., Effect of proline position on the antimicrobial mechanism of buforin II. Peptides 2011, 32 (4), 677-82.
###### 41. Wei, L.; LaBouyer, M. A.; Darling, L. E.; Elmore, D. E., Bacterial Spheroplasts as a Model for Visualizing Membrane Translocation of Antimicrobial Peptides. Antimicrob Agents Chemother 2016, 60 (10), 6350-2.
###### 42. Cutrona, K. J.; Kaufman, B. A.; Figueroa, D. M.; Elmore, D. E., Role of arginine and lysine in the antimicrobial mechanism of histone-derived antimicrobial peptides. FEBS Lett 2015, 589 (24 Pt B), 3915-20.
###### 43. Mihailescu, M.; Sorci, M.; Seckute, J.; Silin, V. I.; Hammer, J.; Perrin, B. S., Jr.; Hernandez, J. I.; Smajic, N.; Shrestha, A.; Bogardus, K. A.; Greenwood, A. I.; Fu, R.; Blazyk, J.; Pastor, R. W.; Nicholson, L. K.; Belfort, G.; Cotten, M. L., Structure and Function in Antimicrobial Piscidins: Histidine Position, Directionality of Membrane Insertion, and pH-Dependent Permeabilization. J Am Chem Soc 2019, 141 (25), 9837-9853.
###### 44. Duan, Y.; Wu, C.; Chowdhury, S.; Lee, M. C.; Xiong, G.; Zhang, W.; Yang, R.; Cieplak, P.; Luo, R.; Lee, T.; Caldwell, J.; Wang, J.; Kollman, P., A point-charge force field for molecular mechanics simulations of proteins based on condensed-phase quantum mechanical calculations. J Comput Chem 2003, 24 (16), 1999-2012.
###### 45. Abraham, M. J.; Murtola, T.; Schulz, R.; Páll, S.; Smith, J. C.; Hess, B.; Lindahl, E., GROMACS: High performance molecular simulations through multi-level parallelism from laptops to supercomputers. SoftwareX 2015, 1-2, 19-25.
###### 46. Lee, L. P.; Tidor, B., Optimization of electrostatic binding free energy. Journal of Chemical Physics 1997, 106 (21), 8681-8690.
###### 47. Kangas, E.; Tidor, B., Optimizing electrostatic affinity in ligand-receptor binding: Theory, computation, and ligand properties. Journal of Chemical Physics 1998, 109 (17), 7522-7545.
###### 48. Radhakrishnan, M. L., Designing electrostatic interactions in biological systems via charge optimization or combinatorial approaches: insights and challenges with a continuum electrostatic framework. Theor. Chem. Acc. 2012, 131 (8).
###### 49. Lee, L. P.; Tidor, B., Optimization of binding electrostatics: Charge complementarity in the barnase-barstar protein complex. Protein Science 2001, 10 (2), 362-377.
###### 50. Lee, L. P.; Tidor, B., Barstar is electrostatically optimized for tight binding to barnase. Nature Structural Biology 2001, 8 (1), 73-76.
###### 51. Ahn, J. S.; Radhakrishnan, M. L.; Mapelli, M.; Choi, S.; Tidor, B.; Cuny, G. D.; Musacchio, A.; Yeh, L. A.; Kosik, K. S., Defining Cdk5 ligand chemical space with small molecule inhibitors of Tau phosphorylation. Chemistry & Biology 2005, 12 (7), 811-823.
###### 52. Armstrong, K. A.; Tidor, B.; Cheng, A. C., Optimal Charges in Lead Progression: A Structure-Based Neuraminidase Case Study. Journal of Medicinal Chemistry 2006, 49 (8), 2470-2477.
###### 53. Minkara, M. S.; Davis, P. H.; Radhakrishnan, M. L., Multiple drugs and multiple targets: An analysis of the electrostatic determinants of binding between non-nucleoside HIV-1 reverse transcriptase inhibitors and variants of HIV-1 RT. Proteins-Structure Function and Bioinformatics 2012, 80, 573-590.
###### 54. Sims, P. A.; Wong, C. F.; McCammon, J. A., Charge optimization of the interface between protein kinases and their ligands. J. Comput. Chem. 2004, 25 (11), 1416-1429.
###### 55. Kangas, E.; Tidor, B., Electrostatic complementarity at ligand binding sites: Application to chorismate mutase. J. Phys. Chem. B 2001, 105 (4), 880-888.
###### 56. Altman, M. D.; Nalivaika, E. A.; Prabu-Jeyabalan, M.; Schiffer, C. A.; Tidor, B., Computational design and experimental study of tighter binding peptides to an inactivated mutant of HIV-1 protease. Proteins-Structure Function and Bioinformatics 2008, 70 (3), 678-694.
###### 57. Li, Q. Progress Toward the Computational Design of Tighter-Binding Antimicrobial Peptides Based on Buforin II. Wellesley College, 2020.
###### 58. Liu, A. Computational Design of Buforin-II-based Antimicrobial Peptides with Improved Electrostatic Interactions with the Cell Membrane. Wellesley College, 2022.
###### 59. Li, L.; Li, C. A.; Sarkar, S.; Zhang, J.; Witham, S.; Zhang, Z.; Wang, L.; Smith, N.; Petukh, M.; Alexov, E., DelPhi: a comprehensive suite for DelPhi software and associated resources. BMC Biophys. 2012, 5, 9.
###### 60. Altman, M. D. Computational ligand design and analysis in protein complexes using inverse methods, combinatorial search, and accurate solvation modeling. Ph.D. Thesis, Massachusetts Institute of Technology, Cambridge, 2006.
###### 61. Drud, A., CONOPT - A GRG code for large sparse dynamic nonlinear optimization problems. Mathematical Programming 1985, 31 (2), 153-191.
###### 62. Corporation, G. D., General Algebraic Modeling System (GAMS). Fairfax, VA, USA, 2021.
###### 63. Nygren, J.; Svanvik, N.; Kubista, M., The interactions between the fluorescent dye thiazole orange and DNA. Biopolymers 1998, 46, 39-51.
###### 64. Lehrer, R. I.; Rosenman, M.; Harwig, S. S. S. L.; Jackson, R.; Eisenhauer, P., Ultrasensitive assays for endogenous antimicrobial polypeptides. J. Immunol. Methods 1991, 137, 167-173.
###### 65. Steinberg, D. A.; Lehrer, R. I., Designer assays for antimicrobial peptides. Disputing the "one-size-fits-all" theory. Methods. Mol. Biol. 1997, 78, 169-186.
###### 66. Jo, S.; Kim, T.; Iyer, V. G.; Im, W., CHARMM-GUI: a web-based graphical user interface for CHARMM. J Comput Chem 2008, 29 (11), 1859-65.
###### 67. Elmore, D. E., Insights into buforin II membrane translocation from molecular dynamics simulations. Peptides 2012, 38 (2), 357-62.
###### 68. Ladokhin, A. S.; Jayasinghe, S.; White, S. H., How to measure and analyze tryptophan fluorescence in membrane properly, and why bother? Anal Biochem 2000, 285 (2), 235-245.
###### 69. Posokhov, Y. O.; Gottlieb, P. A.; Morales, M. J.; Sachs, F.; Ladokhin, A. S., Is lipid bilayer binding a common property of inhibitor cysteine knot ion-channel blockers? Biophys. J. 2007, 93 (4), L20-2.
###### 70. White, S. H.; Wimley, W. C.; Ladokhin, A. S.; Hristova, K., Protein folding in membranes: Determining energetics of peptide-bilayer interactions. Methods Enzymol. 1998, 295, 62-87.
###### 71. Wimley, W. C.; White, S. H., Designing transmembrane a-helices that insert spontaneously. Biochemistry 2000, 39, 4432-4442.
###### 72. Ladokhin, A. S.; Jayasinghe, S.; White, S. H., How to measure and analyze tryptophane fluorescence in membrane properly, and why bother? Anal Biochem 2000, 285 (2), 235-245.
###### 73. Fleming, E.; Maharaj, N. P.; Chen, J. L.; Nelson, R. B.; Elmore, D. E., Effect of lipid composition on buforin II structure and membrane entry. Proteins 2008, 73 (2), 480-491.
###### 74. Epand, R. M.; Epand, R. F., Liposomes as models for antimicrobial peptides. Methods Enzymol 2003, 372, 124-133.
###### 75. Spinella, S. A.; Nelson, R. B.; Elmore, D. E., Measuring peptide translocation into large unilamellar vesicles. J Vis Exp 2012, (59), e3571.
###### 76. Huang, J. Investigating the Effects of Crowding on Antimicrobial Peptides Using Molecular Dynamics Simulations. Wellesley College, 2021.
###### 77. Zhang, Y. X. L.; Radhakrishnan, M. L.; Lu, X. H.; Gross, A. W.; Tidor, B.; Lodish, H. F., Symmetric Signaling by an Asymmetric 1 Erythropoietin: 2 Erythropoietin Receptor Complex. Mol. Cell 2009, 33 (2), 266-274.
###### 78. Shen, Y.; Radhakrishnan, M. L.; Tidor, B., Molecular mechanisms and design principles for promiscuous inhibitors to avoid drug resistance: Lessons learned from HIV-1 protease inhibition. Proteins-Structure Function and Bioinformatics 2015, 83 (2), 351-372.
###### 79. Hallen, M. A.; Martin, J. W.; Ojewole, A.; Jou, J. D.; Lowegard, A. U.; Frenkel, M. S.; Gainza, P.; Nisonoff, H. M.; Mukund, A.; Wang, S.; Holt, G. T.; Zhou, D.; Dowd, E.; Donald, B. R., OSPREY 3.0: Open-source protein redesign for you, with powerful new features. J Comput Chem 2018, 39 (30), 2494-2507.
###### 80. Radhakrishnan, M. L.; Tidor, B., Optimal drug cocktail design: Methods for targeting molecular ensembles and insights from theoretical model systems. J. Chem Inf. Model. 2008, 48 (5), 1055-1073.